build/package.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
If we're just using `sh`, do we want to use this instead? (I always get confused by shebangs, but I've come across more than one SO post in the past where that's the advice. If it works it probably doesn't matter... 😅 )

```suggestion
#!/bin/sh
```

Reply:
For modern systems, this should be an issue. The one particular benefit I can think of in favour of `/usr/bin/env` is that it will use whichever `sh` is available in the user's path (so it doesn't have to be specifically `/bin/sh`). But it's again arguable whether this is a good thing, as it will differ based on the context. I'm inclined to keep it like this.